Rectangle/Utilities/StageUtil.swift

```diff
@@ -14,5 +14,23 @@ class StageUtil {
         
         return stageDefaults.bool(forKey: "GloballyEnabled") && !stageDefaults.bool(forKey: "AutoHide")
     }
+
+    static func stageActuallyVisible() -> Bool {
```

Comment:
Thanks for consistently keeping on top of the feature you introduced! I really appreciate it. A couple notes on this one:

- stageActuallyVisible is a slightly confusing name for the uninitiated, but I'm not picky and it's fine if you want to leave it. Maybe something like stageWindowPresent
- CGWindowListCopyWindowInfo can cause delay for some users on certain macs. Calls to this function typically should be used sparingly. Checking the name/w/h is also potentially fragile. If there's another way we can do this, it might be worth exploring that first. At the very least, we would probably want to only execute this if the defaults for stage manager show it enabled.

Reply:
Apologies for the delay, thanks for the update. You're correct that Rectangle doesn't cache windows. I'm going to go ahead and roll this in. It's probably not too likely that Macs running Ventura will be impacted by one additional call to CGWindowListCopyWindowInfo here, so I'm willing to take the gamble since I can easily troubleshoot back here if needed.